kbase-extension/static/kbase/js/common/html.js

```diff
@@ -555,31 +522,25 @@ define([
                 }
                 return key
                     .replace(/(id|Id)/g, 'ID')
-                    .split(/_/g).map(function (word) {
+                    .split(/_/g).map((word) => {
                     return word.charAt(0).toUpperCase() + word.slice(1);
                 })
                     .join(' ');
             }
+
             function columnValue(row, column) {
-                var rawValue = row[column.key];
+                const rawValue = row[column.key];
                 if (column.format) {
                     return column.format(rawValue);
                 }
-                if (column.type) {
-                    switch (column.type) {
-                        case 'bool':
-                            // yuck, use truthiness
-                            if (rawValue) {
-                                return 'True';
-                            }
-                            return 'False';
-                        default:
-                            return rawValue;
-                    }
+                if (column.type && column.type === 'bool') {
```

Comment:
Fourth time lucky!

I would have refactored this, but (a) time constraints, and (b) no tests.